src/Microsoft.Diagnostics.Runtime/src/DacInterface/DacDataTargetWrapper.cs

```diff
@@ -51,6 +54,10 @@ public DacDataTargetWrapper(DataTarget dataTarget)
             builder = AddInterface(IID_IMetadataLocator, false);
             builder.AddMethod(new GetMetadataDelegate(GetMetadata));
             builder.Complete();
+
+            builder = AddInterface(IID_ICLRRuntimeLocator, false);
```

Comment:
I would wrap this with `if (runtimeBaseAddress > 0) { ... }`.  IE don't expose the interface if we don't have a runtimeBaseAddress.